pxtlib/browserutils.ts

```diff
@@ -166,7 +166,7 @@ namespace pxt.BrowserUtils {
     }
 
     export function isTabletSize(): boolean {
-        return window?.innerWidth < pxt.BREAKPOINT_TABLET;
+        return window?.innerWidth <= pxt.BREAKPOINT_TABLET;
     }
 
     export function isComputerSize(): boolean {
```

Comment:
this function already has it `>=`, so we need to fix that so we never end up with both `isTabletSize() === true` & `isComputerSize() === true` at the same time